dist/build_source_package.sh

```diff
@@ -92,6 +92,31 @@ build_srpm() {
     cp -v "${temp_dir}"/rpm/SRPMS/*.src.rpm "${output_dir}"/rpm/
 }
 
+build_pkgbuild() {
+    check_tools updpkgsums
+
+    mkdir -p "${temp_dir}"/pkg
```

Comment:
Happy to change name. `pkgbuild`? `pacman`? `aur`?

Reply:
Hmm... I'd be happy with any of `pkgbuild`, `pacman`, or `arch`.

Reply:
I'll go with `pkgbuild`